app/javascript/components/patient/Patient.js

```diff
@@ -244,7 +244,7 @@ class Patient extends React.Component {
                   <span className="text-danger">Monitoree is a minor</span>
                   {!this.props.details.head_of_household && this.props.hoh && (
                     <div>
-                      View contact info for head of household:
+                      View contact info for Head of Household:{' '}
```

Comment:
This space was not necessary. Now there are two spaces

Reply:
Hmmmm, you're right. Looks like the actual page renders even if the javascript looks funky on our end. I'll revert the change.

Reply:
@tstrass should be ready for re-review!